packages/web-client/tests/acceptance/card-drop-test.ts

```diff
@@ -0,0 +1,14 @@
+import { module, test } from 'qunit';
+import { visit } from '@ember/test-helpers';
+import { setupApplicationTest } from 'ember-qunit';
+import percySnapshot from '@percy/ember';
+
+module('Acceptance | card-drop', function (hooks) {
+  setupApplicationTest(hooks);
+
+  test('it shows a success message', async function (assert) {
+    assert.expect(0);
```

Comment:
Assertion(s) that might be relevant include the button text and link.

Reply:
I don’t tend to write assertions for static strings, it feels pointless to me… unless I’m misunderstanding?